Fixing StorageEndpointSuffix for AzureUSGovernment

abatishchev opened the pull request:
When I run this command:

> Get-AzureRmEnvironment -Name AzureUSGovernment | Select StorageEndpointSuffix

I'm getting this:

```
StorageEndpointSuffix
---------------------
core.usgovcloudapi.net
```

However this doc contains a dot.

This typo almost caused us an outage in USGov cloud. It didn't thank to a temporal fallback logic to the legacy path.

azurecla commented:
Hi __@seckin206__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

PRmerger commented:
@seckin206 : Thanks for your contribution to the Azure documentation! The author, @sbtron, has been notified to review your proposed change.

sbtron commented:
#sign-off

abatishchev committed:
FIxing StorageEndpointSuffix for AzureUSGovernment

This typo almost caused us an outage in USGov cloud. Didn't thanks to a temporal fallback logic to the legacy path.

When I run this command:

> Get-AzureRmEnvironment -Name AzureUSGovernment | Select StorageEndpointSuffix

I'm getting this:

```
StorageEndpointSuffix
---------------------
core.usgovcloudapi.net
```

However this doc contains a dot.
